Tests/UInt128Tests/UInt128Tests.swift

```diff
@@ -365,57 +372,34 @@ class UInt128UnsignedIntegerTests: XCTestCase {
             previousValue = currentValue
         }
     }
-    func testIndexTypes() {
-        // Test Successor
-        XCTAssertEqual(
-            UInt128(0).successor(), 1,
-            "0.successor() does not equal 1"
-        )
-        XCTAssertEqual(
-            UInt128(UInt64.max).successor(), UInt128(1) << 64,
-            "(UInt64.max).successor() Did Not Cross the Bit Boundary Properly"
-        )
-        XCTAssertEqual(
-            UInt128.max.successor(), 0,
-            "Wraparound From Maximum Value Does Not Equal 0"
-        )
-        // Test Predecessor
-        XCTAssertEqual(
-            UInt128(0).predecessor(), UInt128.max,
-            "Wraparound From 0 Down By 1 Does Not Equal UInt128.max"
-        )
-        XCTAssertEqual(
-            UInt128(UInt128(1) << 64).predecessor(), UInt128(UInt64.max),
-            "(1 << 64).predecessor() Did Not Cross the Bit Boundary Properly"
-        )
-    }
+
 }
 class UInt128StrideableTests: XCTestCase {
     func testAdvancedBy() {
         XCTAssertEqual(
-            UInt128.min.advancedBy(1), UInt128(integerLiteral: 1),
+            UInt128.min.advanced(by: 1), UInt128(integerLiteral: 1),
             "0 Advanced by 1 Does Not Equal 1"
         )
         XCTAssertEqual(
-            UInt128.min.advancedBy(-1), UInt128.max,
+            UInt128.min.advanced(by: -1), UInt128.max,
             "0 Advanced by -1 Does Not Equal UInt128.max"
         )
         XCTAssertEqual(
-            UInt128.max.advancedBy(1), UInt128.min,
+            UInt128.max.advanced(by: 1), UInt128.min,
             "UInt128.max Advanced by 1 Does not Equal UInt128.min"
         )
     }
     func testDistanceTo() {
         XCTAssertEqual(
-            UInt128.min.distanceTo(UInt128(Int.max)), Int.max,
+            UInt128.min.distance(to: UInt128(Int.max)), Int.max,
```

Comment:
This will apply to all of the distance(to:) method tests. I will want to keep distanceTo() tests and implementations until a new major release is cut. As such, I'll want to keep the existing tests, and create additional ones for this instance method. An API should not remove any functionality without warning it's users of this, and the only time that should happen is during a major release update.